GameOfLife/GameOfLifeUI/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using GameOfLifeProcessor;
+using GameOfLifeProcessor.GameBoard;
+using GameOfLifeUI;
+using Pastel;
+using System.Drawing;
+
+//Settings
+var generations = 150;
```

Comment:
ich würde solche Parameter bestenfalls in ein ParameterDto packen, damit ist es leichter zu erweitern ist und die Abhängigkeit nicht zu jeder variable besteht, sondern nur zu dem einen Objekt. ist leichter zu erweitern/anzupassen dann.

Reply:
pending